scripts/build/build/targets.py

```diff
@@ -165,6 +165,7 @@ def BuildHostTarget():
     target.AppendModifier("ipv6only", enable_ipv4=False)
     target.AppendModifier("no-ble", enable_ble=False)
     target.AppendModifier("no-wifi", enable_wifi=False)
+    target.AppendModifier("no-unique-id", disable_unique_id=True)
```

Comment:
Maybe more verbose option like "data-model-no-unique-id" since we already have `data-model-` options.

Reply:
Fixed